### DISCUSSION TITLE
kouadriaek - troubleshooting a dosxyznrc crash

### DISCUSSION THREAD

**Discussion by rtownson** [nrc-cnrc/EGSnrc#1125] [Questions]
@kouadriaek here is a discussion for your question that you posted on https://github.com/nrc-cnrc/EGSnrc/pull/714. Please post further discussion here.

Does the simulation run when you reduce the number of particles? In the images, it looks like you're trying to restart a previous run and continue from where it crashed. Instead, change this so that you're doing a new simulation.

Can you share your input file, so that I can try to reproduce the issue?

**Comment by kouadriaek** [comment]
hi 
this is my inpit file     Le vendredi 3 mai 2024 à 10:58:30 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


@kouadriaek here is a discussion for your question that you posted on #714. Please post further discussion here.

Does the simulation run when you reduce the number of particles? In the images, it looks like you're trying to restart a previous run and continue from where it crashed. Instead, change this so that you're doing a new simulation.

Can you share your input file, so that I can try to reproduce the issue?

**Reply by rtownson** [reply]
Did you intend to attach a file? I don't see it. Please provide the content of the .egsinp file.

**Comment by kouadriaek** [comment]
no the simulation do nt run with small num of un
    Le vendredi 3 mai 2024 à 12:01:08 UTC−7, Abdelkader Kouadri ***@***.***> a écrit :  
 
  
hi 
this is my inpit file     Le vendredi 3 mai 2024 à 10:58:30 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


@kouadriaek here is a discussion for your question that you posted on #714. Please post further discussion here.

Does the simulation run when you reduce the number of particles? In the images, it looks like you're trying to restart a previous run and continue from where it crashed. Instead, change this so that you're doing a new simulation.

Can you share your input file, so that I can try to reproduce the issue?

**Comment by kouadriaek** [comment]
Le vendredi 3 mai 2024 à 12:28:13 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


Did you intend to attach a file? I don't see it. Please provide the content of the .egsinp file.

**Comment by kouadriaek** [comment]
yes i attach my inpust file    Le vendredi 3 mai 2024 à 13:00:34 UTC−7, Abdelkader Kouadri ***@***.***> a écrit :  
 
  

    Le vendredi 3 mai 2024 à 12:28:13 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


Did you intend to attach a file? I don't see it. Please provide the content of the .egsinp file.

**Reply by rtownson** [reply]
The file is not here. Either change the extension to .txt instead of .egsinp in order to attach it, or try just copy/pasting the contents of the .egsinp file into this text box.

**Comment by kouadriaek** [comment]
ELEKTA                                                                           #!GUI1.02H2OICRU521AIRICRU5210.521, 0.01, 0, 0, 0-1, -1, -8, 1-16.10.2, 161-16.10.2, 16100.2, 80.3, 110.2, 10.3, 160.2, 10.3, 320.2, 20.5, 601, 161, 1, 161, 1, 131, 1, 10, 0, 0, 0, 0, 0, 0, 00, 0, 0, 0, 0, 0, 0, 081, 81, 81, 81, 1, 120, 1, 07, 155, 81, 81, 37, 37, 0, 07, 155, 81, 81, 71, 71, 0, 00, 0, 0, 0, 0, 0, 0, 02, 2, 0, 0, 0, 180, 0, 0, 0, 1, 14.2, 100, 100, 52, 0, 2, 0, 0, 0, 0, 0C:/Users/aek/egs_home/BEAM_SYNERGY/SYNERY.egsphsp2100000000, 0, 1000, 31, 72, 100.0, 0, 0, 0, 1, 1.9, 1000, 0, 0, 5, 1, 0, 0 ######################### :Start MC Transport Parameter:  Global ECUT= 0.521 Global PCUT= 0.01 Global SMAX= 5 ESTEPE= 0.25 XIMAX= 0.5 Boundary crossing algorithm= PRESTA-I Skin depth for BCA= 3 Electron-step algorithm= PRESTA-II Spin effects= On Brems angular sampling= Simple Brems cross sections= BH Bound Compton scattering= Off Compton cross sections= default Pair angular sampling= Simple Pair cross sections= BH Photoelectron angular sampling= Off Rayleigh scattering= Off Atomic relaxations= Off Electron impact ionization= Off Photon cross sections= si Photon cross-sections output= Off  :Stop MC Transport Parameter: #########################

    Le vendredi 3 mai 2024 à 13:25:22 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


The file is not here. Either change the extension to .txt instead of .egsinp in order to attach it, or try just copy/pasting the contents of the .egsinp file into this text box.

**Reply by rtownson** [reply]
This is missing the line breaks so I can't really understand it. Can you paste again, but on the lines above and below use three single quotes (```), to denote a code block. Or add in the line breaks manually.

**Comment by kouadriaek** [comment]
Dear Towson 
i send you the file of  dosxyz-win3264 and the messagfes of errors 


    Le lundi 6 mai 2024 à 08:55:40 UTC−7, Reid Townson ***@***.***> a écrit :  
 
 


This is missing the line breaks so I can't really understand it. Can you paste again, but on the lines above and below use three single quotes (```), to denote a code block.
